src/comments.js

```diff
@@ -3,6 +3,7 @@ import { addDays } from "./CommentsColumn/Comment/TopRow/TopRow";
 export default [
   {
     id: 1,
+    src: "https://media-exp1.licdn.com/dms/image/C5103AQES2HWYu834NA/profile-displayphoto-shrink_200_200/0?e=1587600000&v=beta&t=e85cbE830lKu3L4wbpIqVu83o5byIKrhQx8pPpbpPbQ",
```

Comment:
I'd rather name this `profilePicture: "..."` than `src: "..."`. `src` is kind of specific to `<img />` tags, whereas `profilePicture` describes what the property is.